### ISSUE TITLE
oac_check_package: Adding library path to LDFLAGS

### ISSUE THREAD

**Issue by rfeki** [open-mpi/oac#10]
I am working on creating a new component in ompio to support nvidia GPUDirect storage. When creating the config file and using "OAC_CHECK_PACKAGE", the function checks the library path but it does not add -L${check_package_prefix}/lib64 to LDFLAGS as it does for  -L${check_package_prefix}/lib in the function _OAC_CHECK_PACKAGE_GENERIC_PREFIX 

File: oac_check_package.m4
line 624:$2_LDFLAGS=-L${check_package_prefix}/lib
line 628: libdir_prefix=${check_package_prefix}/lib64

I also checked if the variable libdir_prefix is used anywhere else but it's not. Should I make a pull request with this minor change or make a workaround in the new component conf file?

**Comment by jsquyres** [comment]
Ah, are you proposing:

```patch
diff --git a/oac_check_package.m4 b/oac_check_package.m4
index 47d5dd5..6ab607b 100644
--- a/oac_check_package.m4
+++ b/oac_check_package.m4
@@ -625,7 +625,7 @@ disambiguate.])],
                          AC_MSG_RESULT([found -- lib])],
                         [test $check_package_generic_prefix_lib64 -eq 1],
                         [check_package_generic_prefix_happy=1
-                         libdir_prefix=${check_package_prefix}/lib64
+                         $2_LDFLAGS=-L${check_package_prefix}/lib64
                          AC_MSG_RESULT([found -- lib64])],
                         [AC_MSG_RESULT([not found])])])])
 
```

@bwbarrett Does this look right to you?  Or is there a reason that the lib64 stanza is that way?

**Comment by jsquyres** [comment]
Cool.  @rfeki Can you file a PR?

**Comment by rfeki** [comment]
Yes sure

**Comment by raafatfeki** [comment]
PR is filed but I cannot assign reviewers for some reason.

**Comment by bwbarrett** [comment]
Should be fixed now; we'll have to file PRs against OMPI, PMIx, and PRRTE, of course.

**Comment by jsquyres** [comment]
Yeah, we'll have to check which branches in each repo will need this.  E.g., in the `ompi` repo, I think we only need to update `main` and `v5.0.x` (i.e., we're not using the `oac` submodule on the older 4.x branches).

@raafatfeki Can you file submodule updates on each of these repos + branches?  Thanks!

**Comment by rhc54** [comment]
FWIW: PMIx v4.2 branch now uses OAC, and so does PRRTE v3.0

**Comment by jsquyres** [comment]
That will work:

1. Update PMIx oac submodule
2. Update PRRTE oac submodule
3. Update OMPI oac + pmix + prrte submodules

**Comment by bwbarrett** [comment]
Across the three projects, post the PRs in whatever order makes you happy.  There's no dependency that everyone gets the same version of OAC.
